sample-app/src/components/SearchBar.tsx

```diff
@@ -68,29 +83,43 @@ export default function SearchBar({
         placeholder={placeholder}
         screenReaderInstructions={SCREENREADER_INSTRUCTIONS}
         screenReaderInstructionsId={screenReaderInstructionsId}
-        options={autocompleteResponse?.results.map(result => {
-          return {
-            value: result.value,
-            render: () => renderWithHighlighting(result)
-          }
-        }) ?? []}
-        optionIdPrefix='Autocomplete__option'
+        screenReaderText={screenReaderText}
         onSubmit={executeQuery}
-        updateInputValue={value => {
+        onInputChange={(value) => {
           answersActions.setQuery(value);
         }}
-        updateDropdown={() => {
+        onInputFocus={() => {
           autocompletePromiseRef.current = executeAutocomplete();
         }}
         renderButtons={renderSearchButton}
         cssClasses={{
-          optionContainer: 'Autocomplete',
-          option: 'Autocomplete__option',
-          focusedOption: 'Autocomplete__option--focused',
+          dropdownContainer: 'Autocomplete',
           inputElement: 'SearchBar__input',
           inputContainer: 'SearchBar__inputContainer'
         }}
-      />
+      >
+        {
+          options.length > 0 &&
+          <DropdownSection<Option>
+            options={options}
+            optionIdPrefix={`Autocomplete__option-${0}`}
```

Comment:
Why is the zero wrapped in curly braces?

Reply:
Oops that's leftover from my testing